src/components/NewMessageModal.vue

```diff
@@ -34,76 +34,84 @@
 					{{ t('mail', 'Go back') }}
-				</ButtonVue>
-				<ButtonVue type="tertiary"
+				</NcButton>
+				<NcButton type="tertiary"
 					:aria-label="t('mail', 'Send anyway')"
 					@click="onForceSend">
 					{{ t('mail', 'Send anyway') }}
-				</ButtonVue>
+				</NcButton>
 			</template>
 		</EmptyContent>
 		<template v-else>
-			<NcActions class="minimize-button">
-				<NcActionButton :aria-label="t('mail', 'Minimize composer')" @click="onMinimize">
-					<template #icon>
-						<MinimizeIcon :size="20" />
-					</template>
-				</NcActionButton>
-			</NcActions>
-			<Composer ref="composer"
-				:from-account="composerData.accountId"
-				:from-alias="composerData.aliasId"
-				:to="composerData.to"
-				:cc="composerData.cc"
-				:bcc="composerData.bcc"
-				:subject="composerData.subject"
-				:attachments-data="composerData.attachments"
-				:body="composerData.body"
-				:editor-body="convertEditorBody(composerData)"
-				:in-reply-to-message-id="composerData.inReplyToMessageId"
-				:reply-to="composerData.replyTo"
-				:forward-from="composerData.forwardFrom"
-				:send-at="composerData.sendAt * 1000"
-				:forwarded-messages="forwardedMessages"
-				:can-save-draft="canSaveDraft"
-				:saving-draft="savingDraft"
-				:draft-saved="draftSaved"
-				:smime-sign="composerData.smimeSign"
-				:smime-encrypt="composerData.smimeEncrypt"
-				:is-first-open="modalFirstOpen"
-				:request-mdn="composerData.requestMdn"
-				:accounts="accounts"
-				@update:from-account="patchComposerData({ accountId: $event })"
-				@update:from-alias="patchComposerData({ aliasId: $event })"
-				@update:to="patchComposerData({ to: $event })"
-				@update:cc="patchComposerData({ cc: $event })"
-				@update:bcc="patchComposerData({ bcc: $event })"
-				@update:subject="patchComposerData({ subject: $event })"
-				@update:attachments-data="patchComposerData({ attachments: $event })"
-				@update:editor-body="patchComposerData({ editorBody: $event })"
-				@update:send-at="patchComposerData({ sendAt: $event / 1000 })"
-				@update:smime-sign="patchComposerData({ smimeSign: $event })"
-				@update:smime-encrypt="patchComposerData({ smimeSign: $event })"
-				@update:request-mdn="patchComposerData({ requestMdn: $event })"
-				@draft="onDraft"
-				@discard-draft="discardDraft"
-				@upload-attachment="onAttachmentUploading"
-				@send="onSend"
-				@show-toolbar="handleShow" />
+			<NcButton class="maximize-button"
+				type="tertiary-no-background"
+				:aria-label="t('mail', 'Maximize composer')"
+				@click="onMaximize">
+				<template #icon>
+					<MaximizeIcon v-if="!largerModal" :size="20" />
+					<DefaultComposerIcon v-else :size="20" />
+				</template>
+			</NcButton>
+			<NcButton class="minimize-button"
+				type="tertiary-no-background"
+				:aria-label="t('mail', 'Minimize composer')"
+				@click="onMinimize">
+				<template #icon>
+					<MinimizeIcon :size="20" />
+				</template>
+			</NcButton>
 		</template>
+		<Composer ref="composer"
```

Comment:
I don't think the composer should move out of the v-else template. It means the composer would also render when the emptycontent view shows

Reply:
I don't remember moving the template closure on purpose, so I'm assuming it's done by ESLint while i was moving code around. Will push a fixup soon, if you dont have any other comment.